src/executePrettier/executePrettierOnBufferRange.js

```diff
@@ -3,30 +3,32 @@ const _ = require('lodash/fp');
 const prettierEslint = require('prettier-eslint');
 const { allowUnsafeNewFunction } = require('loophole');
 
-const { shouldUseEslint, runLinter } = require('../atomInterface');
-const getPrettierInstance = require('./getPrettierInstance');
-const buildPrettierEslintOptions = require('./buildPrettierEslintOptions');
-const buildPrettierOptions = require('./buildPrettierOptions');
+const { shouldUseEslint, runLinter, getPrettierEslintOptions } = require('../atomInterface');
+const { getCurrentFilePath } = require('../editorInterface');
+const { getPrettierInstance } = require('../helpers');
 const handleError = require('./handleError');
 
 const executePrettier = (editor: TextEditor, text: string) =>
-  getPrettierInstance(editor).format(text, buildPrettierOptions(editor));
+  getPrettierInstance(editor)
```

Comment:
I totally missed that point when you described it in https://github.com/prettier/prettier-atom/issues/256

I feel like "use a config file or the defaults" is too radical.
We have no fallback if there is no config, and it reduces our coverage of use cases.

I understand this for ESLint because there are a lot of rules, but it's manageable for Prettier.

Consider this:

- Not all teams or projects work with config files (I'm in this situation).
- The package will not always be used in a project setting: I could hack on a single file without having to define a `.prettierrc` for example.